transforms/tabular/tabular-thresholding-plugin/README.md

```diff
@@ -0,0 +1,50 @@
+# Tabular Thresholding Plugin
+
+
+This plugin uses three [threshold methods](https://github.com/nishaq503/thresholding.git) to compute threshold values on a user-defined variable and then determines if each label (ROI) is above or below the calculated threshold value. A new feature column will be computed for selected threshold method with the values in  binary format (0, 1) \
+*0* `negative or below threshold`\
+*1* `positive or above threshold`
+
+## Threshold methods
+
+### *1-* False Positive Rate
+It estimates mean and standard deviation of `negControl` values based on the assumption that it follows a single guassian distribution and computes threshold such that the area to the right is equal to a user-defined `falsePositiverate`. Values must range between 0 and 1
+
+### *2-* OTSU
+It computes threshold by using `negControl` and `posControl` values to minimize the weighted variance of these two classes. `numBins` are number of bins to compute histogram of `negControl` and `posControl` values
+
+### *3-* MEAN+Sigma
```

Comment:
Use numbered lists instead of these headings. https://carpentries-incubator.github.io/markdown-intro/06-lists/index.html